plugins/modules/packaging/os/homebrew_cask.py

```diff
@@ -454,18 +466,76 @@ def _current_cask_is_installed(self):
             self.message = 'Invalid cask: {0}.'.format(self.current_cask)
             raise HomebrewCaskException(self.message)
 
-        cmd = [
-            "{brew_path}".format(brew_path=self.brew_path),
-            "cask",
-            "list",
-            self.current_cask
-        ]
+        if self._brew_cask_command_is_deprecated():
+            cmd = [
+                "{brew_path}".format(brew_path=self.brew_path),
+                "list",
+                "--cask",
+                self.current_cask
+            ]
+        else:
+            cmd = [
+                "{brew_path}".format(brew_path=self.brew_path),
+                "cask",
+                "list",
+                self.current_cask
+            ]
+
         rc, out, err = self.module.run_command(cmd)
 
         if rc == 0:
             return True
         else:
             return False
+
+    def _compare_version(self, version1, version2):
+        # convert version string into array of numbers (e.g '1.2.3' => [1,2,3])
```

Comment:
Why not use `LooseVersion` from `distutils.version` (like a lot of other modules and plugins are doing)?

Reply:
ah i didnt know there's this package available. will change to use it. thanks!